packages/cozy-harvest-lib/src/konnector-policies.js

```diff
@@ -9,10 +10,15 @@ const defaultKonnectorPolicy = {
   match: () => true,
   name: 'default',
   fetchExtraOAuthUrlParams: null,
-  refreshContracts: null
+  refreshContracts: null,
+  needsTriggerLaunch: true,
+  needsAccountAndTriggerCreation: true,
+  onLaunch: null,
+  isRunnable: () => true
```

Comment:
`isRunnable` seems to take the `konnector` as arg. I prefer to see it there, same for `match` and else.